Classes/Notification/AbstractNotification.php

```diff
@@ -0,0 +1,157 @@
+<?php
+namespace T3Monitor\T3monitoring\Notification;
+
+/*
+ * This file is part of the t3monitoring extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+use T3Monitor\T3monitoring\Notification\Channel\AbstractNotificationChannel;
+use TYPO3\CMS\Core\Utility\ArrayUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * Class AbstractNotification
+ */
+abstract class AbstractNotification {
+    /**
+     * @var AbstractNotificationChannel[]
+     */
+    protected $channels;
+
+    /**
```

Comment:
wanna move the /** @var array */ syntax? because no description will be used and will save some lines.